install/helm/agones/templates/ping.yaml

```diff
@@ -95,6 +95,10 @@ metadata:
     chart: {{ template "agones.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+{{- if .Values.agones.ping.http.annotations }}
+  annotations:
+{{ toYaml .Values.agones.ping.http.annotations | indent 8 }}
```

Comment:
I believe this should only be indent 4 (i.e. one level deeper than the `annotations` key), but it's only a minor thing.

Reply:
Let me update and see if it would generate proper annotations.